config.json

```diff
@@ -158,7 +158,7 @@
         "slug": "leap",
         "name": "Leap",
         "uuid": "8740c914-f008-4b6b-8e7d-7470e78e6a8e",
-        "prerequisites": [],
+        "prerequisites": ["boolean-logic"],
         "difficulty": 1,
         "topics": [
           "math",
```

Comment:
This is a concept exercise now, needs to be removed in a different PR. Forgot about this when merging #310.

Reply:
As a reminder, we can't move practice to concepts, so:
- This Practice Exercise needs to be marked as deprecated
- The Concept Exercise needs to have a different uuid, etc.
- The exercises need to have different names and slugs to each other (naming the new one differently would be kinder for students)

Reply:
Done.